NeoML/docs/en/Python/tutorials/Linear.ipynb

```diff
@@ -35,43 +35,47 @@
    "id": "dimensional-moderator",
    "metadata": {},
    "source": [
-    "# Linear classificator tutorial"
+    "# Linear classifier"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "grave-reservation",
    "metadata": {},
    "source": [
+    "[Download the tutorial as a Jupyter notebook](https://github.com/neoml-lib/neoml/blob/master/NeoML/docs/en/Python/tutorials/Linear.ipynb)\n",
+    "\n",
+    "In this tutorial, we'll use a NeoML linear classifier to process the [20newsgroups dataset](http://qwone.com/~jason/20Newsgroups/). We'll try out many possible configurations and choose the one that gives best results. NeoML also provides a cross-validation function, which we will use to evaluate each configuration' performance.\n",
```

Comment:
`We'll try out many possible configurations and choose the one that gives best results` is not the best  way to describe what's going on here... It's `Searching for optimal model parameters`.